modules/navier_stokes/test/tests/finite_volume/materials/ergun/ergun.i

```diff
@@ -0,0 +1,426 @@
+# This file simulates flow of fluid in a porous elbow for the purpose of verifying
+# correct implementation of the various different solution variable sets. This input
+# tests correct implementation of the primitive superficial variable set. Flow enters on the top
+# and exits on the right. Because the purpose is only to test the equivalence of
+# different equation sets, no solid energy equation is included.
+
+porosity_left = 0.4
+porosity_right = 0.6
+pebble_diameter = 0.06
+
+mu = 1.81e-5 # This has been increased to avoid refining the mesh
+M = 28.97e-3
+R = 8.3144598
+
+# inlet mass flowrate, kg/s
+mdot = -10.0
+
+# inlet mass flux (superficial)
+mflux_in_superficial = ${fparse mdot / (pi * 0.5 * 0.5)}
+
+# inlet mass flux (interstitial)
+mflux_in_interstitial = ${fparse mflux_in_superficial / porosity_left}
+
+p_initial = 201325.0
+T_initial = 300.0
+rho_initial = ${fparse p_initial / T_initial * M / R}
+
+vel_y_initial = ${fparse mflux_in_interstitial / rho_initial}
+vel_x_initial = 0.0
+
+superficial_vel_y_initial = ${fparse mflux_in_superficial / rho_initial}
+superficial_vel_x_initial = 1e-12
+
+# Computation parameters
+velocity_interp_method = 'rc'
+advected_interp_method = 'upwind'
+
+# ==============================================================================
+# GEOMETRY AND MESH
+# ==============================================================================
+
+[Mesh]
+  [fmg]
+    type = FileMeshGenerator
+    file = 'ergun_in.e'
+  []
+  coord_type = RZ
```

Comment:
an RZ elbow? that s like a nail?

Reply:
You made this test haha. All i did was copy it, gold file and all

Reply:
It used to be ‘insfv_primitive_variable…’, or something like that. Can check when at my comp later

Reply:
ah yeah that rings a bell.  And I think I copied it from another elbow case, not sure if that one was RZ though